packages/tools/addon/services/cardstack-tools.js

```diff
@@ -37,7 +37,26 @@ export default Service.extend({
   activeFields: computed('activeContentItem', 'fields', function() {
     let item = this.get('activeContentItem');
     if (item) {
-      return this.get('fields').filter(f => f.model.content === item.model);
+      return this.get('fields').filter(f => {
+        let activeItemModel = item.model;
+        let activeItemClass = activeItemModel.constructor;
+        let fieldModel = f.model.content;
+        let fieldClass = fieldModel.constructor;
+        if (fieldModel === activeItemModel) {
+          return true;
+        }
+        let isOwnedField = false;
+        activeItemClass.relationships.forEach(([ relationshipDefinition ]) => {
+          let { meta } = relationshipDefinition;
+          let { owned } = meta.options;
+          if (owned) {
+            if (meta.type === fieldClass.modelName) {
+              isOwnedField = true;
```

Comment:
It would be great to jump out of the `forEach` here but that gives an  "unsyntactic break" linting error and there are no other methods on the map backing the relationship definitions that allow to cycle through it. Not a biggie, I think, but worth mentioning.